### PR TITLE
Adding Petaluma  City Of OWRS file for 07-01-2017

### PR DESCRIPTION
**Pull request by ArashSeif**

### CODE REVIEW

File: full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      - 3.95
+      - 4.5
+      - 5.35
+    fixed_drought_surcharge: 0
+    variable_drought_surcharge: 0
+    fixed_wastewater_charge: 0
+    variable_wastewater_charge: 0
+    bill: service_charge+commodity_charge
+  RESIDENTIAL_MULTI:
+    service_charge: 5.74*number_dwelling_unit
```
Comment (ArashSeif) [inline full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs]: Please check if this works.

**Comment by ArashSeif** [comment]
@patwater @christophertull please review
http://cityofpetaluma.net/wrcd/waterrates.html#water

File: full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs (hunk 1 above)
Comment (patwater) [reply full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs]: @christophertull going to punt this one over to you. No rush

Comment (patwater) [reply full_utility_rates/California/Petaluma  City Of - 2158/07-01-2017.owrs]: Note SweetWaterSprings has the same issue: https://github.com/California-Data-Collaborative/Open-Water-Rate-Specification/pull/376/files

**Pull request merged**